### PR TITLE
made translation opt out

### LINKED ISSUE #76: optional translation
would it be of added value to make translations optional for a breadcrumb item?

```php
$breadcrumbs->addItem($user->getUsername(), '', [], false);
```
of course now the translator fails silently and simply outputs the input string, however, it does raise an additional ``translation.WARNING``

### PR DESCRIPTION
**Pull request by wickedOne**
fixes #76

### CODE REVIEW

File: Resources/views/breadcrumbs.html.twig [hunk 1]
```diff
@@ -4,11 +4,11 @@
             {% for b in breadcrumbs %}
                 <li{% if itemClass is defined and itemClass|length %} class="{{ itemClass }}"{% endif %} itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
                     {% if b.url and not loop.last %}
-                        <a href="{{ b.url }}" itemprop="item"{% if linkRel is defined and linkRel|length %} rel="{{ linkRel }}"{% endif %}>
-                    {% endif %}
-                            <span itemprop="name">{{- b.text | trans(b.translationParameters, translation_domain, locale) -}}</span>
-                    {% if b.url and not loop.last %}
-                        </a>
+                    <a href="{{ b.url }}" itemprop="item"{% if linkRel is defined and linkRel|length %} rel="{{ linkRel }}"{% endif %}>
+                        {% endif %}
```
Comment (sampart) [inline Resources/views/breadcrumbs.html.twig]: Can you tidy up the indentation here please?  The `if` and `endif` blocks should be at the same level of indentation.

Comment (wickedOne) [inline Resources/views/breadcrumbs.html.twig]: oops...
done

**Comment by sampart** [comment]
Thanks again for this.  Let me know if you need a release making.

**Comment by wickedOne** [comment]
you're welcome.
yes a release would be nice, thanks!

**Comment by sampart** [comment]
Done: https://github.com/whiteoctober/BreadcrumbsBundle/releases/tag/1.2.3

**Pull request merged**